crates/brainfuck_vm/src/bin/brainfuck_vm.rs

```diff
@@ -45,9 +45,6 @@ fn main() -> Result<(), MachineError> {
     tracing::info!("Provide inputs separated by linefeeds: ");
     bf_vm.execute().unwrap();
     if args.trace {
-        if args.pad_trace {
```

Comment:
Also remove the pad_trace argument in the `Args` struct (cannot comment on it for the review): L23-L34 of this file